app/forms/openid_connect_authorize_form.rb

```diff
@@ -207,6 +207,8 @@ def extra_analytics_attributes
     {
       client_id: client_id,
       redirect_uri: result_uri,
+      scope: scope,
+      acr_values: acr_values,
```

Comment:
these two are arrays, which can be tricky to query in Cloudwatch. Should we consider joining to strings?
```suggestion
      scope: scope.sort.join(' '),
      acr_values: acr_values.sort.join(' '),
```

Reply:
yes please!

Reply:
have this staged and will push once i see what this most recent CI build does